### PR TITLE
[DE] Add `Graveyard`

### PR DESCRIPTION
**Pull request by Bloeckchengrafik**
It's another Addition to the german Wiki!

### CODE REVIEW

File: wiki/Graveyard/de.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+stub: true
+no_native_review: true
+tags:
+  - dead
+---
+
+# Graveyard
+
+Der **Graveyard** (auf Deutsch: Friedhof) ist eine Unterform von [Beatmaps](/wiki/Beatmap), welche nicht mehr von ihren Erstellern (Mappern) beachtet werden.
```
Comment (Phippe) [inline wiki/Graveyard/de.md:10]: I wouldn't translate _subforum_ as _Unterform_ (maybe this was an inadvertent mistake anyway?). The literal translation sounds alright to me (_Unterforum_), but I think _Unterkategorie_ would be better.

Also, it's probably clear what _Ersteller_ means, so the _Mappern_ in brackets feels a bit redundant.

File: wiki/Graveyard/de.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+tags:
+  - dead
+---
+
+# Graveyard
+
+Der **Graveyard** (auf Deutsch: Friedhof) ist eine Unterform von [Beatmaps](/wiki/Beatmap), welche nicht mehr von ihren Erstellern (Mappern) beachtet werden. 
+Solche Maps zählen nicht zum maximalen Uploadlimit des Benutzers. Der [Beatmap-Host](/wiki/Beatmap/Beatmap_host) kann diese "wiederbeleben", 
+indem er diese bearbeitet. Hierfür wird ein freier Map-Slot benötigt. Danach wird die Map in 
+["Work in Progress (in Bearbeitung)"](/wiki/Beatmap/Category#work-in-progress-and-pending) verschoben.
```
Comment (Phippe) [inline wiki/Graveyard/de.md:13]:
```suggestion
indem er diese bearbeitet. Hierfür wird ein freier Map-Slot benötigt. Danach wird die Map in den
["Work in Progress (in Bearbeitung)"](/wiki/Beatmap/Category#work-in-progress-and-pending)-Status zurückversetzt.
```

**Review by Phippe — COMMENTED** [review]
There doesn't exist consensus about whether to use the translated or the English term for graveyard on [crowdin](https://crowdin.com/translate/osu-web/all/en-de?filter=advanced&value=12&sort_method=1&sort_ascending=0#q=graveyard). I'm leaning towards translating it, but I guess both are alright (just make sure to update the "wrong" term on crowdin after merging this for consistency).

If you decide to translate it, you could still add something like "eng.: Graveyard" in brackets for more clarity.

Everything here is more of a suggestion than being absolutely necessary :D

**Comment by TicClick** [comment]
don't want to disappoint you, but this stub is being removed in #5984. please put your effort in translating /wiki/Beatmap/Category instead

**Pull request closed without merging**